src/scan.ts

```diff
@@ -189,6 +189,11 @@ async function run() {
     const GITHUB_TOKEN = core.getInput("github_token");
     const GITHUB_PR_ID = github.context.payload.pull_request?.number;
 
+    const USE_BAZEL = core.getInput("use_bazel");
```

Comment:
`core.getBooleanInput` can be used here instead of `core.getInput`. `getInput` returns a string value for the input.

```suggestion
    const USE_BAZEL = core.getBooleanInput("use_bazel");
```